app/src/views/search-engine/index.js

```diff
@@ -44,7 +47,48 @@ const SearchEngine = () => {
         setLoading={setLoading}
         loading={loading}
       />
+      <Chip
+        size='small'
+        label={`${articles.length} resultados`}
+        className={classes.resultsChip}
+      />
       <ArticlesList articles={articles} />
+      {articles.length === 0 &&
```

Comment:
If a query returns no articles the info card will be displayed?

Reply:
Yes, that's correct!

Reply:
I forgot to respond to this, but won't this be super confusing for the user

Reply:
+1 it'd be nice to display a no results found or something so that users don't think your app is just taking forever

Reply:
I actually added a small results count right below the search bar!! So yeah, you actually get some feedback in the results count.